src/components/Comments.jsx

```diff
@@ -27,24 +60,49 @@ function Comments() {
     return <p>error!</p>;
   } else if (loading) {
     return <p>Loading comments for this article! Please wait a moment.</p>;
+  } else if (comments.length === 0) {
+    return (
+      <h3>
+        {" "}
+        There are no comments yet! Would you like to write one?
+        <button id="white-button" onClick={popwithAddComment}>
+          add a comment
+        </button>
+      </h3>
+    );
   } else {
-    if (comments.length === 0) {
-      return <h3> There are no comments yet! Would you like to write one?</h3>;
-    } else {
-      return (
-        <ul>
-          {comments.map((comment) => {
-            return (
-              <li key={comment.comment_id} className="Listcomments">
-                <p>comment by {comment.author}</p>
-                <p>Votes: {comment.votes}</p>
-                <p>{comment.body}</p>
-              </li>
-            );
-          })}
-        </ul>
-      );
-    }
+    return (
+      <ul>
+        <button id="red-button" onClick={popwithAddComment}>
+          add a comment
+        </button>
+        <button id="blue-button" onClick={refreshComments}>
+          refresh comments
+        </button>
+        {addComment ? (
+          <AddNewComment comments={comments} setComments={setComments} />
+        ) : null}
+        {comments.map((comment) => {
+          return (
+            <li key={comment.comment_id} className="Listcomments">
+              <p>comment by {comment.author}</p>
+              <p>
+                Votes: {comment.votes} {"   "}
+                {user !== comment.author ? null : (
+                  <button
+                    id="blue-button"
+                    onClick={() => popwithdeleteComment(comment.comment_id)}
+                  >
+                    delete your comment
+                  </button>
+                )}
+              </p>
+              <p>{comment.body}</p>
+            </li>
+          );
+        })}
+      </ul>
+    );
```

Comment:
Great implementation of a delete button, I like that you've only make the delete button available for the logged in user's comments 👍